src/js/profile/profile_header.js

```diff
@@ -136,9 +140,34 @@ export class Profile_header extends Observable {
         $('.location__facilities_loading').addClass('hidden');
     }
 
+    downloadAllFacilities = () => {
+        this.api.loadAllPoints(this.profileId, geography.code)
+            .then((data) => {
+                const wb = XLSX.utils.book_new();
+                const fileName = 'Facilities-' + geography.code + '.xlsx';
+                data.results.forEach((r) => {
+                    const sheetName = this.getSheetName(r.category);
+                    let rows = [];
+                    r.features.forEach((f) => {
+                        let row = {};
```

Comment:
You could use es6 features to make this more concise:

```js
let { properties: { name, data } } =  f
let row = {...data, name }
```

And you can reduce that even more with
```js
let rows = r.features.map((f) => {
  let { properties: { name, data } } =  f
  return {...data, name }
})
```